.ci/tests/integration-oauth2/cases/py-download-function-legacy/verify.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```

Comment:
this need to be added in https://github.com/streamnative/function-mesh/blob/master/.ci/tests/integration/e2e.yaml#L141

Reply:
fixed